samples/snippets/src/main/java/com/example/spanner/ObservabilityLab.java

```diff
@@ -189,9 +193,12 @@ private void performMonotonicallyIncreasingWrite(
         mutations.add(
             Mutation.newInsertBuilder("Message")
                 .set("msg_id")
-                .to(Instant.now().toString() + i + Thread.currentThread().getId())
+                // This will add the incremental value and be thread safe. While re-running need to
+                // make sure that we delete the Ids from Table  or create fresh database as this
+                // will always start from 1
+                .to(msdId.getAndIncrement())
                 .set("body")
-                .to(String.format("test-body-%s", i))
+                .to(LOREM_IPSUM)
```

Comment:
Added this here based on Hailong comment to keep some bigger value in table. 
In case of Mutliparticipant test I did not keep this as I found having small value there changes AVG_Participant to 2 in 12 min easily